src/textual/widgets/_select.py

```diff
@@ -305,6 +305,51 @@ def __init__(
         self._value = value
         self._setup_variables_for_options(options)
 
+    @classmethod
+    def from_values(
+        cls,
+        values: Iterable[SelectType],
+        *,
+        prompt: str = "Select",
+        allow_blank: bool = True,
+        value: SelectType | NoSelection = BLANK,
+        name: str | None = None,
+        id: str | None = None,
+        classes: str | None = None,
+        disabled: bool = False,
+    ):
```

Comment:
This seems to be missing a return type hint.

Reply:
I've made a change in the latest commit, but I'm not entirely sure it is correct.  I've not used type hints with class methods before, so please let me know if what I did was wrong.